project/src/main/puzzle/puzzle-money-hud.gd

```diff
@@ -0,0 +1,48 @@
+extends Control
+## Tween which shows/hides the money label which appears after a puzzle.
```

Comment:
This comment is inaccurate.

Reply:
Fixed comment, `Shows/hides the money label after a puzzle.`.

Also added comments to the rest of the script.